File: tests/nodejs/cases/case_300_container_handoff_global_09.js
/**
 * Task case 300: container_handoff_global_09 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case300ContainerHandoffGlobal09(input) {
  const taskName = 'container_handoff_global_09';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const box = { deep: { value: payload } };
  // ESCAPE: nested indirection retained globally.
  retainedCache.set('nested', box.deep.value);
  return 'ok';
}

module.exports = { case300ContainerHandoffGlobal09 };

File: tests/go/cases/case_300_container_handoff_global_09.go
package escape_tests

var retainedCase300 = []map[string]string{}

func Case300ContainerHandoffGlobal09(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "container_handoff_global_09",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "container_handoff_global_09:" + raw,
	}
	box := map[string]map[string]string{"v": payload}
	// ESCAPE: nested container handoff retained globally.
	retainedCase300 = append(retainedCase300, box["v"])
	return "ok"
}
